sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonIndexer.java

```diff
@@ -64,6 +67,22 @@ ProjectLevelSymbolTable projectLevelSymbolTable() {
     return projectLevelSymbolTable;
   }
 
+  void removeFile(InputFile inputFile) {
+    String packageName = packageNames.get(inputFile.uri());
+    packageNames.remove(inputFile.uri());
+    PythonFile pythonFile = SonarQubePythonFile.create(inputFile);
+    projectLevelSymbolTable.removeModule(packageName, pythonFile);
```

Comment:
I think we could just pass the file name here so that we don't need to create `PythonFile`

Reply:
Indeed, if you don't mind, I'll address this in my next PR, as I have a significant refactoring of the class on the way and it'll make the rebase a bit easier :)